docs/api-spec/post-votes-by-comment.md

```diff
@@ -0,0 +1,16 @@
+method | endpoint | definition | 
+-------| -------- | ---------- |
+post   | /api/comment/**{commentID}**/votes | submit vote to a comment
+
+# url params
+`commentID`
+
+# request
+```
+n/a
+```
+
+# response
+```
+{ "result": "ok" }
```

Comment:
For consistency and verbosity, please don't use single-line examples. See https://github.com/platf0rm/platform/pull/9/commits/b997e368b5aeba61829257987a01defda3938f08#diff-03db1675cc9d339a9c38c4af5f3c5a05 as an example.